core/src/main/java/bio/terra/pearl/core/dao/survey/SurveyDao.java

```diff
@@ -24,11 +24,20 @@ public Optional<Survey> findByStableIdWithMappings(String stableId, int version)
         return surveyOpt;
     }
 
+    public List<Survey> findByStableIdNoContent(String stableId) {
+        List<Survey> surveys = jdbi.withHandle(handle ->
```

Comment:
since this is the second place we're fetching stuff without content, it might be worth pulling out a helper `getNonContentColumns()` method that returns a list of Strings for the columns these methods are interested in.